docs/quickstart_guide/index.rst

```diff
@@ -0,0 +1,245 @@
+
+
+Install Lewis
+=============
+
+The recommended way to install Lewis is via PyPI and using a virtual environment. This guide assumes you have Python and Pip installed and in your PATH.
+
+Install ``virtualenv`` and create a virtual environment. On Windows, the activate script will be located elsewhere and can be executed directly, without ``source``. This step is optional and may be skipped:
+
+::
+
+    $ pip install virtualenv
+    $ virtualenv myenv
+    $ source myenv/bin/activate
+
+Lewis can be installed with Pip using a single command:
+
+::
+
+    $ pip install lewis
+    $ lewis --version
+
+
+Run the Motor Example
+=====================
+
+Once Lewis is installed, you can use it to start some of the example devices it ships with. 
+
+You can see which devices are available by just executing Lewis without parameters:
+
+::
+
+    $ lewis
+    Please specify a device to simulate. The following devices are available:
+        julabo
+        chopper
+        linkam_t95
+
+Some additional, simpler examples are located in the ``lewis.examples`` module. You can tell Lewis which module to scan for devices using the ``-k`` parameter:
+
+::
+
+    $ lewis -k lewis.examples
+    Please specify a device to simulate. The following devices are available:
+        dual_device
+        simple_device
+        timeout_device
+        modbus_device
+        example_motor
+
+For this guide, we will launch the example_motor:
+
+::
+
+    $ lewis -r :10000 -k lewis.examples example_motor
+    INFO lewis.DeviceBase: Creating device, setting up state machine
+    INFO lewis.Simulation: Changed cycle delay to 0.1
+    INFO lewis.Simulation: Changed speed to 1.0
+    INFO lewis.Simulation: Starting simulation
+    INFO lewis.ControlServer: Listening on 0.0.0.0:10000
+    INFO lewis.AdapterCollection: Connecting device interface for protocol 'stream'
+    INFO lewis.ExampleMotorStreamInterface.StreamServer: Listening on 0.0.0.0:9999
+
+The example motor is a TCP Stream device and listens on port 9999 on all adapters by default.
+
+The ``-r`` parameter is optional and specifies an ``adapter:port`` pair for the "Control API" to listen on. Here it listens on port 10000 on all adapters. We will see more on that in following sections.
+
+
+Connect to Motor via Telnet
+===========================
+
+With the last command from the previous section still running, open another terminal window.
+
+Since the example motor conveniently uses CRLF line terminators, we can use telnet to talk to it:
+
+::
+
+    $ telnet localhost 9999
+    Trying 127.0.0.1...
+    Connected to localhost.
+    Escape character is '^]'.
+
+You're now connected to the TCP Stream interface of the example motor device. It understands the following commands:
+
+=======  =============
+Command  Meaning
+=======  =============
+S?       get status
+P?       get position
+T?       get target
+T=%f     set target
+H        stop movement
+=======  =============
```

Comment:
You can also extract what these commands are? Is it overload to say how you would do that at this point? Maybe introduce the `-i` argument?

Reply:
Yes, that's a good idea. I think I'll just add it as a suggestion after the table though, rather than replacing the table with the output of `-i`, since this table is a bit more compact.